vpdq/setup.py

```diff
@@ -6,50 +6,77 @@
 import sys
```

Comment:
I was never happy with the way that this `setup.py` worked, glad you could get a crack at it.